libsolidity/codegen/ir/IRGenerator.h

```diff
@@ -115,6 +119,9 @@ class IRGenerator
 	langutil::EVMVersion const m_evmVersion;
 	OptimiserSettings const m_optimiserSettings;
 
+	std::set<ASTNode const*> m_creationFunctionList;
+	std::set<ASTNode const*> m_runtimeFunctionList;
```

Comment:
```suggestion
	std::set<ASTNode const*> m_deployedFunctionList;
```

Reply:
Wouldn't it be better to keep generated functions in the IRGenerationContext and retrieve the set from there?